docs/community/reporting_security_concerns.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Reporting Security Concerns
+sidebar_position: 50
+---
+
+# Security Policy
+
+Thank you for taking the time to report a security vulnerability. 
+We would like to investigate every report thoroughly.
+
+## Reporting a Vulnerability
+
+To report a security vulnerability, please contact one or more (ORAS maintainers)[https://github.com/oras-project/community/blob/main/OWNERS.md] via email or Slack with the vulnerability details. 
```

Comment:
I think this was going to be updated to use the github security here advisor.

Reply:
@TerryHowe it has been updated now, sorry for taking so long.